root/static/scripts/edit/URLCleanup.js

```diff
@@ -1317,6 +1317,19 @@ const CLEANUPS = {
       return {result: false};
     },
   },
+  'dnb': {
+    match: [
+      new RegExp('^(https?://)?([^/]+\\.)?d-nb\\.info', 'i'),
+      new RegExp('^(https?://)?([^/]+\\.)?dnb\\.de', 'i'),
+    ],
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?(?:[^\/]+\.)?dnb\.de\/opac\.htm\?.*\bquery=nid%3D(\d+).*$/, 'http://d-nb.info/gnd/$1');
+    },
+    validate: function (url) {
+      return {result: /^http:\/\/d-nb\.info\/gnd\/(\d+)$/.test(url)};
+    },
```

Comment:
`/gnd` links (`nid` queries) seem to be exclusively for artists, so we should probably validate that it's only used for artists. See how we do it elsewhere on this file :) You'll also want to change the test to use `only_valid_entity_types: ['artist']`

Also, they also have `idn` queries (https://portal.dnb.de/opac.htm?method=simpleSearch&cqlMode=true&query=idn%3D1227621485), that can be cleaned up to links like http://d-nb.info/1227621485 and which work now, but this would make invalid (https://musicbrainz.org/url/a4eaabd0-55ba-4918-970f-6eeb48aa91d7 for example). Can you make this so that it cleans those up too and allows them for releases? Add a test for them too :)